desktop/src/main/java/bisq/desktop/main/account/register/DisputeResolverRegistrationView.java

```diff
@@ -0,0 +1,270 @@
+                return new ListCell<>() {
+                    final Label label = new AutoTooltipLabel();
+                    final ImageView icon = ImageUtil.getImageViewById(ImageUtil.REMOVE_ICON);
+                    final Button removeButton = new AutoTooltipButton("", icon);
+                    final AnchorPane pane = new AnchorPane(label, removeButton);
+
+                    {
+                        label.setLayoutY(5);
+                        removeButton.setId("icon-button");
+                        AnchorPane.setRightAnchor(removeButton, 0d);
+                    }
+
+                    @Override
+                    public void updateItem(final String item, boolean empty) {
+                        super.updateItem(item, empty);
+                        if (item != null && !empty) {
+                            label.setText(LanguageUtil.getDisplayName(item));
+                            removeButton.setOnAction(e -> onRemoveLanguage(item));
+                            setGraphic(pane);
+                        } else {
+                            setGraphic(null);
+                        }
+                    }
+                };
+            }
+        });
+
+        languageComboBox = FormBuilder.addComboBox(gridPane, ++gridRow);
+        languageComboBox.disableProperty().bind(model.registrationEditDisabled);
+        languageComboBox.setPromptText(Res.get("shared.addLanguage"));
+        languageComboBox.setConverter(new StringConverter<>() {
+            @Override
+            public String toString(String code) {
+                return LanguageUtil.getDisplayName(code);
+            }
+
+            @Override
+            public String fromString(String s) {
+                return null;
+            }
+        });
+        languageComboBox.setOnAction(e -> onAddLanguage());
+
+        final Tuple2<Button, Button> buttonButtonTuple2 = add2ButtonsAfterGroup(gridPane, ++gridRow,
+                Res.get("account.arbitratorRegistration.register"), Res.get("account.arbitratorRegistration.revoke"));
+        Button registerButton = buttonButtonTuple2.first;
+        registerButton.disableProperty().bind(model.registrationEditDisabled);
+        registerButton.setOnAction(e -> onRegister());
+
+        Button revokeButton = buttonButtonTuple2.second;
+        revokeButton.setDefaultButton(false);
+        revokeButton.disableProperty().bind(model.revokeButtonDisabled);
+        revokeButton.setOnAction(e -> onRevoke());
+
+        final TitledGroupBg titledGroupBg = addTitledGroupBg(gridPane, ++gridRow, 2,
+                Res.get("shared.information"), Layout.GROUP_DISTANCE);
+
+        titledGroupBg.getStyleClass().add("last");
+
+        Label infoLabel = addMultilineLabel(gridPane, gridRow);
+        GridPane.setMargin(infoLabel, new Insets(Layout.TWICE_FIRST_ROW_AND_GROUP_DISTANCE, 0, 0, 0));
+        infoLabel.setText(Res.get("account.arbitratorRegistration.info.msg"));
```

Comment:
This info text is not generic enough for both mediator and arbitrator

Reply:
Ah good point. Wanted to ask @ripcurlx how it is for translators if we change key of string. I fear they will see he old key removed and need to add the new one again. That's why I left it currently, but I will add a TODO.